app/components/admin_edit/admin_edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<% content_for @title do %><%= @title %><% end %>
```

Comment:
we eventually decided just to put the components and .rb files in the root components directory - im actually a little suprised this is working as previously the rb file needed to be in the root and the template in its own directory, whcih is why we moved. puzzling!